content/en/docs/components/trainer/legacy-v1/user-guides/job-scheduling.md

```diff
@@ -204,9 +204,39 @@ spec:
           restartPolicy: OnFailure
 ```
 
+### KAI Scheduler
+
+Install KAI Scheduler in your cluster by following [the installation steps](https://github.com/NVIDIA/KAI-Scheduler/tree/main?tab=readme-ov-file#installation). Upon installation, KAI Scheduler automatically creates a default-queue for workload submission. KAI Scheduler's podgrouper component automatically detects Training Operator jobs and creates PodGroup resources for gang-scheduling.
+
+See the [batch workload guide](https://github.com/NVIDIA/KAI-Scheduler/blob/main/docs/batch/README.md#pytorchjob) for an example. Once you submit, KAI scheduler automatically;
+- Detects your jobs
+- Creates PodGroup resources
+- Assigns them to default-queue for gang-scheduling
+
+#### Optional: Custom Queues
+
+For advanced resource management with multiple queues:
+1. Create queues by following the [queue configuration guide](https://github.com/NVIDIA/KAI-Scheduler/blob/main/docs/queues/README.md)
+2. Assign jobs to specific queues by adding the kai.scheduler/queue label:
+
+```diff
+...
+    metadata:
+      name: my-training-job
+      labels:
++        kai.scheduler/queue: my-queue-name
+...
+```
+
+
+
+#### Elastic Training
+KAI Scheduler supports elastic workloads, where you can specify minimum (gang threshold) and maximum pod counts. If the number of running pods falls below the minimum threshold, the entire workload is evicted. KAI Scheduler intelligently manages pod roles, prioritizing eviction of non-leader pods when possible.
```

Comment:
It would be interesting to investigate Elastic Training for TrainJob and KAI Scheduler: https://github.com/kubeflow/trainer/blob/master/pkg/apis/trainer/v1alpha1/trainingruntime_types.go#L219